GREEDY_WEB/ClientApp/components/ImageUpload.tsx

```diff
@@ -0,0 +1,49 @@
+﻿import * as React from 'react';
+import ImageUploader from 'react-images-upload';
+import axios from 'axios';
+
+interface Props {
+    updateItemList: any;
+}
+
+export class ImageUpload extends React.Component<Props> {
+    constructor(props) {
+        super(props);
+        this.state = {
+            itemList: []
+        };
+    }
+
+    _handleImageChange(e) {
+        e.preventDefault();
+
+        let reader = new FileReader();
+        let file = e.target.files[0];
+
+        this.state = {
+            itemList: []
+        };
```

Comment:
why we need this?

Reply:
If I remeber correctly this is because of typescript. Javascript lets you do anything with undefined variables but typescript show an error. So I declered it (even if I declared it as an empty array it is still defined)